aether-common-module/aether/common/kernel/utils.py

```diff
@@ -145,16 +145,17 @@ def get_data(url):
     return results
 
 
-def submit_to_kernel(submission, mapping_id, submission_id=None):
+def submit_to_kernel(submission, submission_fk, submission_id=None):
```

Comment:
Why not just call it `mappingset_id` instead of `submission_fk`? it got me confused. We also have `project` as a `fk`

Reply:
Yesterday it was `mapping_id`, now is `mappingset_id` and tomorrow... ?

I want to reduce kernel data model dependency in the external modules.

But I don't mind to change this back :woman_shrugging:

Reply:
I agree, this also got me confused. The move from `mapping` to `mappingset` is a major change, so I think it's OK to reflect that in the code rather than to abstract away from it. It's certainly not something that should happen very often.